### DISCUSSION TITLE
Display git tags

### DISCUSSION THREAD

**Discussion by Sayanel01** [starship/starship#5602] [Q&A]
Hi everybody,
I'm looking for a way to display latest tag along with current branch name. I can do that with the "Git commit" module if I set `tag_disabled=false` and `only_detached=false`

So tag is show, and not only in detached head state. But this interferes with the default behavior that I like, which is show commit name (alone) when in detached head. 

What I want is "if detached head, show commit name in bold green, if not detached head show latest tag in regular white" 

I could tweak the `format` string to but that seems tedious. Is there a better way to keep both behaviors? 

Thanks

**Comment by awebeer256** [comment]
I think you want something like #4923 or #5602.

I also want them 😔.
